### PR TITLE
TW-503: Apply group chat setting UI

### PR DESCRIPTION
**Pull request by nqhhdev**
- Issue: #503 

- Resolved

### MOBILE 

https://github.com/linagora/twake-on-matrix/assets/99852347/aaeeaab8-e5ba-45a2-8f04-d1d71c12a7bc

### WEB

https://github.com/linagora/twake-on-matrix/assets/99852347/4cd8738d-26e4-486b-a9ff-94da2d6c144a

### CODE REVIEW

File: lib/widgets/app_bars/searchable_app_bar_style.dart [hunk 1]
```diff
@@ -5,8 +5,36 @@ import 'package:flutter/cupertino.dart';
...
+          isFullScreen: isFullScreen,
+        ),
+      );
 
-  static double appBarHeight(BuildContext context) => 56;
+  static double appBarHeight(
+    BuildContext context, {
+    bool? isFullScreen = true,
+  }) =>
+      isFullScreen == true ? 56 : 112;
```
Comment (sherlockvn) [inline lib/widgets/app_bars/searchable_app_bar_style.dart:23]: remove `context`

**Comment by sherlockvn** [comment]
> * Issue: [Apply Group Chat settings UI #503](https://github.com/linagora/twake-on-matrix/issues/503)
> * Resolved

Demo please

File: lib/widgets/app_bars/searchable_app_bar_style.dart (hunk 1 above)
Comment (nqhhdev) [reply lib/widgets/app_bars/searchable_app_bar_style.dart:20]: done

File: lib/pages/chat_details/chat_details.dart [hunk 2]
```diff
@@ -351,6 +381,82 @@ class ChatDetailsController extends State<ChatDetails> {
...
+      barrierDismissible: false,
+      useSafeArea: false,
+      useRootNavigator: PlatformInfos.isMobile ? false : true,
+      builder: (context) {
+        return SlotLayout(
+          config: <Breakpoint, SlotLayoutConfig>{
+            const WidthPlatformBreakpoint(
+              begin: ResponsiveUtils.minDesktopWidth,
+            ): SlotLayout.from(
+              key: invitationSelectionMobileAndTabletKey,
```
Comment (hoangdat) [inline lib/pages/chat_details/chat_details.dart:396]: wrong key

File: lib/pages/chat_details/chat_details.dart [hunk 3]
```diff
@@ -351,6 +381,82 @@ class ChatDetailsController extends State<ChatDetails> {
...
+
+  List<ChatDetailsActions> chatDetailsActionsButton() => [
+        if (responsive.isDesktop(context)) ChatDetailsActions.addMembers,
+        ChatDetailsActions.mute,
+        ChatDetailsActions.search,
+        ChatDetailsActions.more,
+      ];
+
+  List<ChatDetailsPageModel> chatDetailsPages() => [
+        if (responsive.isDesktop(context))
```
Comment (hoangdat) [inline lib/pages/chat_details/chat_details.dart:423]: use pageView for all platforms

File: lib/pages/chat_details/chat_details_header_view_web.dart
Comment (hoangdat) [inline lib/pages/chat_details/chat_details_header_view_web.dart:1]: change file name

**Review by hoangdat — APPROVED** [review]

**Pull request merged**